doc/neosnippet.txt

```diff
@@ -216,6 +216,15 @@ g:neosnippet#enable_completed_snippet
 		prototype.
 
 		The default value is 0.
+		
+					*g:neosnippet#arguments_are_optional*
+g:neosnippet#arguments_are_optional
+		If this variable is not 0, neosnippet expanded function
```

Comment:
This feature is for completed snippet only.
I think it should be described in the documentation.